### DISCUSSION TITLE
Prefix env var names with directus_ ?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13348] [Feature Requests]
The env variable names are fairly standard names and may clash with names that are used by other systems which could be running on the system if not run in a container. Prefixing with something like `directus_` will prevent this.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
